external-crates/move/crates/move-compiler/src/sui_mode/linters/mod.rs

```diff
@@ -123,6 +126,12 @@ pub fn known_filters() -> (Option<Symbol>, Vec<WarningFilter>) {
             LINTER_DEFAULT_DIAG_CODE,
             Some(COLLECTION_EQUALITY_FILTER_NAME),
         ),
+        WarningFilter::code(
```

Comment:
We probably need to build a separate list for these since we will want to include them with a flag